src/models/StyledNativeComponent.js

```diff
@@ -95,8 +96,23 @@ export default (constructWithOptions: Function) => {
       }
     }
 
+    setNativeProps(nativeProps: Object) {
+      this.root.setNativeProps(nativeProps)
+    }
+
+    generateRef() {
+      const { innerRef } = this.props
+      return (component: any) => {
```

Comment:
When you modify this to be a method, it'd be great to call this `instance` or `node`, or sth, since it's not a component :)